<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>950</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ip地址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ip"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_user"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_con">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_src">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_loc">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>90</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_src">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>140</y>
     <width>450</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_loc">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>140</y>
     <width>450</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_put">
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get">
   <property name="geometry">
    <rect>
     <x>754</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_record">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>570</y>
     <width>950</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>641</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上级目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_flush">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
